### PR TITLE
SPOI-10488-azure-deployment-template

### PR DESCRIPTION
**Pull request by yogidevendra**
1. Azure deployment template, script for RTS installation.

### CODE REVIEW

File: utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        "Standard_G2",
+        "Standard_G3",
+        "Standard_G4",
+        "Standard_G5",
+        "Standard_H8",
+        "Standard_H16",
+        "Standard_H8m",
+        "Standard_H16m",
+        "Standard_H16r*",
+        "Standard_H16mr*"
```
[the remark this answers is not in the document]
Comment (yogidevendra) [reply utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json]: Not possible in json.

File: utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      "apiVersion": "2015-03-01-preview",
+      "properties": {
+        "marketPlaceIdentifier": "EmptyNode",
+        "computeProfile": {
+          "roles": [
+            {
+              "name": "edgenode",
+              "targetInstanceCount": 1,
+              "hardwareProfile": {
+                "vmSize": "Standard_D3_v2"
```
[the remark this answers is not in the document]
Comment (yogidevendra) [reply utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json]: done

File: utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+            "roles": [
+              "edgenode"
+            ]
+          }
+        ],
+        "uninstallScriptActions": [],
+        "httpsEndpoints": [
+          {
+            "subDomainSuffix": "dat",
+            "destinationPort": 9090,
```
[the remark this answers is not in the document]
Comment (yogidevendra) [reply utils/azure/hdinsight-with-Datatorrent-RTS/azuredeploy.json]: done.

**Comment by chaithu14** [comment]
LGTM

**Pull request merged**